### PR TITLE
Progress bar Fix and 100 DAI deposit fix

### LINKED ISSUE #45: Approved DAI display is wrong
**This has to display 100 DAI**

Bildschirmfoto 2020-03-17 um 12 13 38

### PR DESCRIPTION
**Pull request by simonweniger**
this one resolves #45 #44 #37 
with @MaxStalker 

- fixed the progress bar view
- fixed the wrong displayed DAI after form submission

### CODE REVIEW

File: packages/frontend/src/components/FundingSubmission.js [hunk 1]
```diff
@@ -323,11 +323,13 @@ export default class FundingSubmission extends Component {
...
-                      <Form.TextArea
+                      <p> Add a link to your Proposal from <a href="https://discourse.rosebud.fund/">our Forum </a> </p>
+                      <Form.Input
                         name="description"
-                        label="Description"
-                        placeholder="Type here"
+                        placeholder="https://discourse.rosebud.fund/c/proposals/myproposal"
                         rows={15}
+                        type="url"
+                        pattern="https://discourse.rosebud.fund/.*"
```
Comment (johba37) [inline packages/frontend/src/components/FundingSubmission.js]: in a regular expression special characters should be escaped, like this: `^https:\/\/discourse\.rosebud\.fund\/.*`
also `^` will require the input to start with the url

File: packages/frontend/src/components/FundingSubmission.js [hunk 2]
```diff
@@ -323,11 +323,13 @@ export default class FundingSubmission extends Component {
...
                   <Grid.Column>
                     <Segment className="blurred box">
-                      <Form.TextArea
+                      <p> Add a link to your Proposal from <a href="https://discourse.rosebud.fund/">our Forum </a> </p>
+                      <Form.Input
                         name="description"
-                        label="Description"
-                        placeholder="Type here"
+                        placeholder="https://discourse.rosebud.fund/c/proposals/myproposal"
                         rows={15}
```
Comment (johba37) [inline packages/frontend/src/components/FundingSubmission.js]: why 15 rows, isn't 1 enough for a url?

**Review by johba37 — COMMENTED** [review]
2 small remarks

File: packages/frontend/src/components/FundingSubmission.js (hunk 2 above)
Comment (simonweniger) [reply packages/frontend/src/components/FundingSubmission.js]: yeah forgot to delet that

**Comment by simonweniger** [comment]
Here are all results:

Bildschirmfoto 2020-03-19 um 11 39 09
Bildschirmfoto 2020-03-19 um 11 38 49
Bildschirmfoto 2020-03-19 um 11 39 02

**Review by johba37 — APPROVED** [review]
🍍

**Pull request merged**